### ISSUE TITLE
Installation of jellyfin media player does not work: Error: Nothing matches com.github.iwalton3.jellyfin-media-player in remote flathub

### ISSUE THREAD

**Issue by patrickmerz** [flathub/com.github.iwalton3.jellyfin-media-player#2]
Hi,

I tried to install the jellyfin media player via flatpak, but get the following error:
Error: Nothing matches com.github.iwalton3.jellyfin-media-player in remote flathub

What I did:
I installed flatpak as recommended on raspberry pi 4 (see https://www.flatpak.org/setup/Raspberry%20Pi%20OS/):
- sudo apt install flatpak
- sudo flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo
Then reboot of raspi

Then I tried to install jellyfin media player:
- flatpak install flathub com.github.iwalton3.jellyfin-media-player

Note that the directories 

'/var/lib/flatpak/exports/share'
'/home/pi/.local/share/flatpak/exports/share'

are not in the search path set by the XDG_DATA_DIRS environment variable, so
applications installed by Flatpak may not appear on your desktop until the
session is restarted.

Looking for matches...
Error: Nothing matches com.github.iwalton3.jellyfin-media-player in remote flathub

**Comment by hfiguiere** [comment]
>  I installed flatpak as recommended on raspberry pi 4 (see https://www.flatpak.org/setup/Raspberry%20Pi%20OS/):

Do you run a 64-bits (aarch64) OS on the Raspberry Pi?

**Comment by hfiguiere** [comment]
Just to clarify, and it's mentionned on the setup page you list: if you run the 32-bits OS (the default from Raspberry Pi foundation, as they hide the 64-bits version) you are unlikely to have proper support. This application flatpak was created after armv7 was dropped so it doesn't exist.

**Comment by iwalton3** [comment]
I believe there’s an ARM64 build but regardless I wouldn’t bother trying to run it on RPI. MPV doesn’t support hardware acceleration so the media won’t play very well. Use Jellyfin for Kodi instead.

**Comment by patrickmerz** [comment]
@hfiguiere:
> Do you run a 64-bits (aarch64) OS on the Raspberry Pi?
No, I am still running on 32bit, because the 64bit port is still unstable...
@iwalton3:
Yes, I already switched to Kodi as client with Jellyfin for Kodi plugin.

Thanks for the answers and clarification.

**Comment by hfiguiere** [comment]
> @hfiguiere:
> 
> > Do you run a 64-bits (aarch64) OS on the Raspberry Pi?
> 
> No, I am still running on 32bit, because the 64bit port is still unstable...

Then that's why.

*hint* the 64-bits OS work great. It is not unstable.
